Makefile

```diff
@@ -192,6 +200,47 @@ $(FLG_TSTS): $(DST_TEST)/%.tested.flag: $(SRC_TST)/%.ttl $(wildcard $(SRC_ONT)/*
            exit 1; \
          }
 
+## Fuseki:
+.PHONY: fuseki-start
+fuseki-start: build ## Start Fuseki server with the ontology and examples loaded in it
+	@echo "${COLOR_CYAN}🚀 starting ${COLOR_GREEN}Fuseki${COLOR_RESET} server"
+	@if [ "$$(docker ps -q -f name=${DEPLOYMENT_FUSEKI_CONTAINER})" ]; then \
+      echo "${COLOR_CYAN}❌ container ${COLOR_GREEN}${DEPLOYMENT_FUSEKI_CONTAINER}${COLOR_RESET} already running"; \
+	  exit 1; \
+	fi
+	@docker run \
+	  --rm \
+	  -d \
+	  --name ${DEPLOYMENT_FUSEKI_CONTAINER} \
+	  -p ${DEPLOYMENT_FUSEKI_PORT}:${DEPLOYMENT_FUSEKI_PORT} \
+	  -v `pwd`/shiro.ini:/fuseki/shiro.ini \
+	  -e -JVM_ARGS=${DEPLOYMENT_FUSEKI_JVM_ARGS} \
+	  ${DOCKER_IMAGE_FUSEKI}
+	@sleep 1
+	@echo "${COLOR_CYAN}⏱️ waiting for REST API to be ready...${COLOR_RESET}"
+	@timeout ${DEPLOYMENT_FUSEKI_STARTUP_TIMEOUT} sh -c 'until $$(curl --output /dev/null --silent --head --fail http://localhost:${DEPLOYMENT_FUSEKI_PORT}/$$/ping); do \
```

Comment:
I think this command is not installed by default on MacOS systems, we may check its presence

Reply:
🤔 good point. I make the change to include [timeout](https://man7.org/linux/man-pages/man1/timeout.1.html) and [md5sum](https://man7.org/linux/man-pages/man1/md5sum.1.html) as well (which was missing too).

Reply:
Both can be installed on MacOS systems with `brew install coreutils`

Reply:
Done. See: https://github.com/okp4/ontology/pull/116/commits/9c10eaddb27f32f5d0c16e40dba1e0a4137ef05c.

Reply:
Done by https://github.com/okp4/ontology/commit/b06a66409911e321ff6765eb2e5ed88d2edba631.